components/python/dbus-python/Makefile

```diff
@@ -33,20 +35,71 @@ COMPONENT_LICENSE_FILE=	COPYING
 
 PATH=$(PATH.gnu)
 
-PYTHON_VERSION = 3.5
-PYTHON_VERSIONS = 3.5
+PYTHON_VERSIONS = 3.5 3.7 3.9
```

Comment:
This can be expressed as
PYTHON_VERSIONS= $(PYTHON3_VERSIONS)

Reply:
I've changed that macro.